UPGRADING.rst

```diff
@@ -10,6 +10,16 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#3497 Fix: API Gateway log is missing important fields
+======================================================
+
+Upgrading the format of API Gateway logs requires Terraform to run the
+CloudWatch log group provisioner script. Run::
+
+    (cd terraform && terraform taint aws_cloudwatch_log_group.indexer \
+    && terraform taint aws_cloudwatch_log_group.service) && make deploy
```

Comment:
If `taint` supports multiple resources in one invocation, I would like us to make use of that.

More importantly, I would like us to explore the possibility of avoiding the tainting altogether. As written now, the REST API resource depends on the log group resource. Try inserting a null resource in between those two, with a hard-coded trigger that is simply a number. Move the provisioner to the null resource. The goal is to run the provisioner whenever the number (essentially a version) changes OR after the log group resource is created (for example after tainting it). Test both scenarios.